main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -196,26 +196,39 @@ private void loadDeviatedFixedRouteFeature(GtfsFeedContainer feedContainer) {
   }
 
   private boolean hasAtLeastOneTripWithAllFields(GtfsFeedContainer feedContainer) {
-    Optional<GtfsTableContainer<?>> optionalStopTimeTable =
-        feedContainer.getTableForFilename(GtfsStopTime.FILENAME);
-    if (optionalStopTimeTable.isPresent()) {
-      for (GtfsEntity entity : optionalStopTimeTable.get().getEntities()) {
-        if (entity instanceof GtfsStopTime) {
-          GtfsStopTime stopTime = (GtfsStopTime) entity;
-          return stopTime.hasTripId()
-              && stopTime.tripId() != null
-              && stopTime.hasLocationId()
-              && stopTime.locationId() != null
-              && stopTime.hasStopId()
-              && stopTime.stopId() != null
-              && stopTime.hasArrivalTime()
-              && stopTime.arrivalTime() != null
-              && stopTime.hasDepartureTime()
-              && stopTime.departureTime() != null;
-        }
-      }
-    }
-    return false;
+    return feedContainer
+        .getTableForFilename(GtfsStopTime.FILENAME)
+        .map(table -> (GtfsStopTimeTableContainer) table)
+        .map(GtfsStopTimeTableContainer::byTripIdMap)
+        .map(
+            byTripIdMap ->
+                byTripIdMap.keySet().stream()
```

Comment:
Why do we need to know the key (tripId)? 
Shouldn't we just obtain the lists of lists of GtfsStopTime with byTripIdMap.asMap().values()? We know that each list of GtfsStopTime obtained this way contains all the GtfsStopTime for a given tripId, no?
Not sure if it's better though.
Your version does the job, so feel free to ignore.